### ISSUE TITLE
Allow users to apply a specific string and bypass the rest of the HideSettingsPages UI.

### ISSUE THREAD

**Issue by DrewNaylor** [DrewNaylor/HideSettingsPages#30]
By allowing this, people can just apply a string they copied and pasted from elsewhere online or a text document they made as a backup for whatever their preferred Settings app page configuration is. This could be in the File menu, possibly as "File>Directly apply key value...". Clicking on it will open a dialog similar to the Windows "Run" dialog but instead of running a program, whatever they type into the box will be written to the proper Registry key value when they click "Apply". There will then be a messagebox asking the user if they're sure they want to apply the key value (which can be turned off in the Options window once it exists, or maybe just a dropdown on the menubar.) If they choose to apply the key value, UAC will show up asking for the admin password and apply the key value with hsp_registry-helper.exe.

**Comment by DrewNaylor** [comment]
I could either do it like this, or I could have that window be its own EXE file. I wanted it to say "Launch minimal GUI..." and have the other window be called "HideSettingsPages Minimal GUI," but it may be better to have "Directly apply key value..." and have the other window be called "HideSettingsPages Minimal" and tell the user they can type in a specific key value to apply without having to click the checkboxes they want to use. I'm not sure though.

**Comment by DrewNaylor** [comment]
For the foreseeable future, it'll just be a dialog box in the main HideSettingsPages EXE file. Menubar title: "Directly Apply Key Value". File menu entry: "Directly apply key value...". Shortcut: "Ctrl+D".

**Comment by DrewNaylor** [comment]
Window mockup:

![window mockup](https://user-images.githubusercontent.com/15517419/34920093-227edd18-f93b-11e7-9a2b-94e80b8cd075.PNG)

Current code for mockup:

```vbnet

'MinimalGUI - Minimal GUI for HideSettingsPages
'(MinimalGUI also known as HideSettingsPages Minimal GUI)
'Copyright (C) 2017  Drew Naylor
'Microsoft Windows and all related words are copyright
'and trademark Microsoft Corporation.
'Any other companies mentioned own their respective copyrights/trademarks.
'(Note that the copyright years include the years left out by the hyphen.)
'
'This file is part of HideSettingsPages
'(Program is also known as "Hide Windows 10 Settings App Pages.")
'
'HideSettingsPages Minimal GUI is free software: you can redistribute it and/or modify
'it under the terms of the GNU General Public License as published by
'the Free Software Foundation, either version 3 of the License, or
'(at your option) any later version.
'
'HideSettingsPages Minimal GUI is distributed in the hope that it will be useful,
'but WITHOUT ANY WARRANTY; without even the implied warranty of
'MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
'GNU General Public License for more details.
'
'You should have received a copy of the GNU General Public License
'along with HideSettingsPages Minimal GUI.  If not, see <http://www.gnu.org/licenses/>.



Imports System.Windows.Forms
Public Class aaformMinimalGUI
    Private Sub buttonApply_Click(sender As Object, e As EventArgs) Handles buttonApply.Click
        If textboxFullValue.TextLength > 0 Then
            ' Tell the registry helper app to apply the key value in the Registry.
            Dim proc As New ProcessStartInfo
            proc.FileName = My.Application.Info.DirectoryPath & "\hsp_registry-helper.exe"
            proc.Arguments = "/apply " & textboxFullValue.Text
            proc.Verb = "runas"
            Try
                Process.Start(proc)
                ' We have to catch this exception
                ' in case the user clicks "No" in the UAC
                ' dialog. Otherwise, we get an error
                ' that says that the operation was
                ' canceled by the user.
            Catch ex As System.ComponentModel.Win32Exception
            End Try
        Else

            ' If there's nothing in the textbox, complain.
            MessageBox.Show("Textbox is empty.")

        End If

    End Sub

    Private Sub buttonUndo_Click(sender As Object, e As EventArgs) Handles buttonUndo.Click
        ' Tell the registry helper app to delete the key value in the Registry.
        Dim proc As New ProcessStartInfo
        proc.FileName = My.Application.Info.DirectoryPath & "\hsp_registry-helper.exe"
        proc.Arguments = "/undo "
        proc.Verb = "runas"
        Try
            Process.Start(proc)
            ' We have to catch this exception
            ' in case the user clicks "No" in the UAC
            ' dialog. Otherwise, we get an error
            ' that says that the operation was
            ' canceled by the user.
        Catch ex As System.ComponentModel.Win32Exception
        End Try
    End Sub

    Private Sub buttonCancel_Click(sender As Object, e As EventArgs) Handles buttonCancel.Click
        Me.Close()
    End Sub

    Private Sub linklabelAbout_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles linklabelAbout.LinkClicked
        ' Show an About window and give a link to th license.
        MessageBox.Show("HideSettingsPages Minimal GUI version 0.1" & vbCrLf &
                        "Copyright 2017 Drew Naylor. Licensed under Gnu GPL v3+." & vbCrLf &
                        "See <http://www.gnu.org/licenses/> for the full license." & vbCrLf &
                        "Any copyrights or trademarks owned by Microsoft or other" & vbCrLf &
                        "companies or people are owned by them.")
    End Sub
End Class

```

**Comment by DrewNaylor** [comment]
This is now implemented as of 47d4970131154533dff50a81a852157b5d3e3cff. Prior commits have the feature fully implemented, but this one is better for control sizing.
